src/pyvesync/vesynckitchen.py

```
@@ -205,7 +205,7 @@ def status_request(self, json_cmd: dict) -> None:  # pylint: disable=R1260
                 self.cook_set_time = cook.get('cookSetTime', self.cook_set_time)
                 self.cook_set_temp = cook.get('cookSetTemp', self.cook_set_temp)
                 self.current_temp = cook.get('currentTemp', self.current_temp)
-                self.temp_unit = cook.get('tempUnit', self.temp_unit)
+                self.temp_unit = str(cook.get('tempUnit', self.temp_unit))
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)